### ISSUE TITLE
Running train.py just stops when trying to finetune train pretrained model on custom dataset? 

### ISSUE THREAD

**Issue by DerpyMan101** [NVIDIA/tacotron2#323]
After making my own dataset and following the same structure as 
 
[https://github.com/NVIDIA/tacotron2/blob/master/filelists/ljs_audio_text_val_filelist.txt](url) and
[https://github.com/NVIDIA/tacotron2/blob/master/filelists/ljs_audio_text_train_filelist.txt](url)

adding my own filelist files and editing the hparams to their location, other than that I changed the batch size to 16.

I downloaded the tacotron2_statedict.pt pretrained model and imported it and then ran: `python train.py --output_directory=outdir --log_directory=logdir -c tacotron2_statedict.pt --warm_start` and it outputed:

`WARNING:tensorflow:
The TensorFlow contrib module will not be included in TensorFlow 2.0.
For more information, please see:
  * https://github.com/tensorflow/community/blob/master/rfcs/20180907-contrib-sunset.md
  * https://github.com/tensorflow/addons
  * https://github.com/tensorflow/io (for I/O related ops)
If you depend on functionality not listed there, please file an issue.

FP16 Run: False
Dynamic Loss Scaling: True
Distributed Run: False
cuDNN Enabled: True
cuDNN Benchmark: False
Epoch: 0

C:\Users\Graham\Desktop\tacotron2-master>`

It just ends and I tried to run it again with tensorboard open but got nothing. 

The files I am using are 16khz .wav files which I also changed in the hparams to 16khz.

**Comment by tdplaza** [comment]
Because you have passed the -c param, the checkpoint_path variable (in train function of train.py) should be not None. 
If it is not None and warn_start, then the warm_start_model function should be run, and prints out the message  `print("Warm starting model from checkpoint '{}'".format(checkpoint_path))` (or through the assertion error if the checkpoint_path is not file.

But it does not, then either checkpoint_path is none or the warn_start is False, or both.

Put some print command after `print("Epoch: {}".format(epoch))` line to check where is it stopped?
after getting where is it stopped, print some variables value to check are they normal or not.

**Comment by rafaelvalle** [comment]
What error do you get?

**Comment by graham-eisele** [comment]
I didn't get any errors when I run it, it just doesn't start training.

**Comment by Thien223** [comment]
then you should put some `print()` commands to check which line is error holder.

**Comment by Text2-m** [comment]
I also encountered this problem, have you solved it？

**Comment by matko123** [comment]
@DerpyMan101 did you perhaps solve the problem? Having the same problem. Thanks in advance.

**Comment by jinny1208** [comment]
I had the same problem, but when I waited for some time, it started training.
